docs/make-react-component.md

```diff
@@ -0,0 +1,48 @@
+@function react-view-models.makeReactComponent makeReactComponent
+@parent react-view-models 2
+
+@description Creates a React Component out of a [can-component].
+
+
+@signature `makeReactComponent( displayName, CanComponent )`
+
+Convert a [can-component] into a React Component.
+
+`makeReactComponent()` takes 2 arguments. The first (optional) is the displayName of the ReactComponent. The second argument is a CanComponent constructor function. The `makeReactComponent()` function returns a React Component which can then be imported and used in any react component or render function as usual.
```

Comment:
It seems redundant to have this documentation and then the `@param` documentation below. I would merge them together.